packages/backend/src/open_payments/receiver/service.test.ts

```diff
@@ -0,0 +1,217 @@
+import { IocContract } from '@adonisjs/fold'
```

Comment:
This test looks quite a lot different that the previous OpenPaymentsClientService one, there are some duplicated code but IMO more straightforward